zebra/rt_netlink.c

```diff
@@ -2814,12 +2830,15 @@ netlink_vxlan_flood_update_ctx(const struct zebra_dplane_ctx *ctx, int cmd,
 			       void *buf, size_t buflen)
 {
 	struct ethaddr dst_mac = {.octet = {0}};
+	int zebra2proto = dplane_ctx_get_type(ctx) ?
+		zebra2proto(dplane_ctx_get_type(ctx)) : RTPROT_ZEBRA;
```

Comment:
yeah, no, you can't really do that.
how about just 'proto'? and how about making it a little easier to read?

```
int proto = RTPROT_ZEBRA;

if (dplane_ctx_get_type(ctx) != 0)
    proto = zebra2proto(dplane_ctx_get_type(ctx));
```

Reply:
ok. I follow your commendations. lets proceed.